packages/dashboard-backend/src/devworkspaceClient/services/kubeConfigApi.ts

```diff
@@ -75,8 +75,31 @@ export class KubeConfigApiService implements IKubeConfigApi {
           this.getServerConfig(),
         );
 
-        // if -f ${kubeConfigDirectory}/config is not found then sync kubeconfig to the container
-        const kubeConfig = this.setNamespaceInContext(this.kubeConfig, namespace);
+        // check if the kubeconfig is already mounted
+        if (container.volumeMounts?.some(vm => vm.mountPath.includes(kubeConfigDirectory))) {
```

Comment:
?
```suggestion
        if (container.volumeMounts?.some(vm => vm.mountPath === kubeConfigDirectory)) {
```

Reply:
I just committed it. At first, I thought that the include would be better in case of deep path kubeconfig but didn't think about the possible case where the path without any link could include this kind of path